source/_integrations/history_stats.markdown

```diff
@@ -178,6 +178,20 @@ Here, last Monday is _today_ as a timestamp, minus 86400 times the current weekd
 
 {% endraw %}
 
+**Next 4pm **: ends today at 00:00, lasts 30 days. Easy one.
```

Comment:
- 4pm does not appear to be part of the example below.
- Please remove "Easy one.", it doesn't add anything and what may be easy for you, might not be easy for someone else.